code/__DEFINES/mobs.dm

```diff
@@ -297,14 +297,16 @@
 //Used as an upper limit for species that continuously gain nutriment
 #define NUTRITION_LEVEL_ALMOST_FULL 535
 
-//Charge levels for Ethereals, in joules.
+// The standard charge all other Ethereal charge defines are scaled against.
+#define STANDARD_ETHEREAL_CHARGE (1 * STANDARD_CELL_CHARGE)
```

Comment:
This should be STANDARD_BATTERY_CHARGE else it just reopens #86985 cause ethereals won't make a dent in draining battery from the apc.

You said so yourself in the comments

Reply:
I also tested this pr myself. It works.
It just doesn't drain it significantly, and that's a knowingly made compromise for now.
This was decided when we had to pick between two prs, one that aligned Ethereal discharge rate with `STANDARD_BATTERY_CHARGE` (#88771), or one which aligned Ethereal charge with `STANDARD_CELL_CHARGE` (#88877), and we picked the latter.

Besides that, due to how the procs are refactored in this pr, the same things that caused it to error there no longer apply here. Furthermore, the reason it errored and blocked it entirely was due to a mismatch in charge defines, which is already addressed by the standardization here.